templates/zerver/api/get-profile.md

```diff
@@ -35,7 +35,9 @@ zulip(config).then((client) => {
 
 ## Arguments
 
-This endpoint takes no arguments.
+**Note**: The following arguments are all URL query parameters.
```

Comment:
@orientor hmm, should we instead be communicating about parameter types as declared in the OpenAPI data in a more structured way?